docs/rules/sort-array-includes.md

```diff
@@ -146,9 +146,9 @@ export default [
       'perfectionist/sort-array-includes': [
         'error',
         {
-          type: 'natural',
-          order: 'asc',
-          spread-last: true,
+          "type": 'natural',
+          "order": 'asc',
+          "spread-last": true,
```

Comment:
Could you fix formatter issue?
In the case of `type` and `order` we don't need quotes, in the case of `spread-last` the quotes should be single quotes.

Reply:
Now should be fine.